src/utils/hooks/withConfirmation/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "action_apply": "Proceed",
+  "context_unsaved-changes-warning": "You have unsaved changes in this query. Do you want to proceed?"
```

Comment:
It's better to write in the Query editor or tab, "this query" isn't very clear thing when I am already in diagnostics or some other not query tab (e.g. Top Queries, History or Saved)

<img width="1394" alt="Screenshot 2024-11-14 at 16 12 39" src="https://github.com/user-attachments/assets/e22c6594-f839-41ee-afbc-c3279060f654">